### PR TITLE
[AngularUniversalBundle] compile also webpack extra config files specified in configuration

### PR DESCRIPTION
**Pull request by massimilianobraglia**
This PR aims to permit users to specify in configuration the webpack extra configuration files, compile and merge into general webpack configuration.

Can you give it a look, @alekitto?

### CODE REVIEW

File: src/Compiler/Webpack/ConfigurationBuilder.js [hunk 1]
```diff
@@ -56,6 +59,14 @@ class ConfigurationBuilder {
         };
 
         const host = new NodeJsAsyncHost();
+
+        let webpackExtraConfigs = [];
+        for (const config of this._webpackConfigPaths) {
+            if (fs.existsSync(config)) {
```
Comment (alekitto) [inline src/Compiler/Webpack/ConfigurationBuilder.js]: If the file does not exist an exception should be thrown

File: src/DependencyInjection/Configuration.js [hunk 2]
```diff
@@ -24,6 +24,11 @@ class Configuration extends implementationOf(ConfigurationInterface) {
                     .isRequired()
                     .info('The tsconfig.json location')
                 .end()
+                .arrayNode('webpack_extra_configs')
+                    .scalarPrototype().end()
+                    .defaultValue(['%kernel.project_dir%/extra-webpack.config.js'])
```
Comment (alekitto) [inline src/DependencyInjection/Configuration.js]: Empty array as default value. Do not assume any extra configuration.

**Review by alekitto — CHANGES_REQUESTED** [review]
Looks good. Just left a couple of comments.

File: src/Compiler/Webpack/ConfigurationBuilder.js (hunk 1 above)
Comment (massimilianobraglia) [reply src/Compiler/Webpack/ConfigurationBuilder.js]: 👍

File: src/DependencyInjection/Configuration.js (hunk 2 above)
Comment (massimilianobraglia) [reply src/DependencyInjection/Configuration.js]: :+1:

**Review by alekitto — APPROVED** [review]
LGTM

**Comment by alekitto** [comment]
Thank you @massimilianobraglia

**Pull request merged**